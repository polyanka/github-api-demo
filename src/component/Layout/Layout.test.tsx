import { shallow } from 'enzyme';
import * as React from 'react';

import { Layout } from './index';

describe('<Layout /> ', () => {
  const children = <p>Hello</p>;

  it('renders correctly Layout component', () => {
    const wrapper = shallow(<Layout children={children} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('check render children', () => {
    const wrapper = shallow(<Layout children={children} />);
    expect(wrapper.find('p').text()).toEqual('Hello');
  });
});
