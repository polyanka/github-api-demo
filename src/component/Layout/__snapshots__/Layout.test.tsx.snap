// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout />  renders correctly Layout component 1`] = `
<Fragment>
  <Header />
  <WithStyles(ForwardRef(Container))>
    <Styled(MuiBox)
      mt={4}
    >
      <p>
        Hello
      </p>
    </Styled(MuiBox)>
  </WithStyles(ForwardRef(Container))>
</Fragment>
`;
